rwgame/GameConfig.cpp

```diff
@@ -155,6 +155,7 @@ bool GameConfig::parseConfig(
                     } else {
                       success = false;
                       RW_MESSAGE(e.what());
+                      return;
                     }
                 }
```

Comment:
Also: Why don't we at least warn when the value could not be parsed? It should at least spit a warning if the value could not be parsed properly.